image/Makefile

```diff
@@ -4,7 +4,9 @@ CLI_DIR:=$(CURDIR)/../../cli
 VERSION?=0.0.0-dev
 STATIC_VERSION=$(shell ../static/gen-static-ver $(ENGINE_DIR) $(VERSION))
 DOCKER_HUB_ORG?=docker
+ARCH=$(unamae -m)
 ENGINE_IMAGE?=engine-community
+TAG_HUB_ORG?=
```

Comment:
Where/how does this get used? I can't find another instance of this variable.

Reply:
It doesn't get used.
Thank you!